services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import Q
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet[Movie]:
+    movies = Movie.objects.all()
+    filter_kwargs = {}
+
+    if genres_ids:
+        filter_kwargs["genres__id__in"] = genres_ids
```

Comment:
movies = movies.filter(genres__id__in=genres_ids) if you write in this way , u can skip creating dictionary and filter_kwargs

Reply:
yup but I wanted to make it in a more `pythonic` way, as seems for me, by not repeating code.

Reply:
Yup, but for larger amount of args/kwargs I could make function which will summarize any amount of them into the dict, which will work just fine. And in case it's a studying project. as for me, it's great idea to realize some templates for further idea although I totally agree that in exact case it's not necessary.
Sure, when getting id's distinct is useless, but as I said, was thinking about template more, than on current task, I'll refactor this.

Reply:
Idea was in a template for future works with larger amount of args/kwargs which I could unpack and easily implement through code, I agree it's not needed in exact case. Same for `distinct()`